src/intrprtr.c

```diff
@@ -140,20 +103,13 @@ static void INTERPRETER_PROFILE_HOOK(int ignoreLevel)
 **
 **  'PopVoidObj' returns the  top element from the values  stack and pops it.
 **  It is an error if the stack is empty but not if the top element is void.
-**
-**  Since interpreters  can nest, there can   be more than one  values stack.
-**  The bottom  element of each values stack is the 'StackObj' which was
-**  active when the current interpreter was started and which will be made
-**  active again when the current interpreter will stop.
 */
-/* TL: Obj             IntrState; */
```

Comment:
As explained above, we don't need `STATE(IntrState)` anymore, so this comment on its predecessor (it used to be a simple global variable) can be removed, too.